### DISCUSSION TITLE
How do I link to the SDL2 x64 static library (sdl2:x64-windows-static) using vcpkg?

### DISCUSSION THREAD

**Discussion by roussec** [microsoft/vcpkg#23196] [Q&A]
hi,

it seems that although I installed the static library of SDL2 vcpkg searches in installed/x64-windows instead of installed/x64-windows-static. I can work around this by setting a system environment variable (SDL2_DIR) which points to the correct variable but this is not a good solution. Am I doing something wrong?

Thank you

**Comment by Osyotr** [comment]
Did you installed it in manifest mode or via `vcpkg install sdl2:x64-windows-static`? You should prefer first approach and set `VCPKG_TARGET_TRIPLET` cache variable to `x64-windows-static`.

**Reply by roussec** [reply]
I installed it via `vcpkg install sdl2:x64-windows-static`. Do you have a link where I can read up on manifest mode ? Thanks

**Reply by Osyotr** [reply]
https://vcpkg.io/en/docs/examples/manifest-mode-cmake.html
